components/ide/jetbrains/launcher/main.go

```diff
@@ -98,6 +99,7 @@ func main() {
 	// supervisor refer see https://github.com/gitpod-io/gitpod/blob/main/components/supervisor/pkg/supervisor/supervisor.go#L961
 	shouldWaitBackendPlugin := os.Getenv("GITPOD_WAIT_IDE_BACKEND") == "true"
 	debugEnabled := os.Getenv("SUPERVISOR_DEBUG_ENABLE") == "true"
+	preferToolbox := os.Getenv("GITPOD_PREFER_TOOLBOX") == "true"
```

Comment:
We have to use env so that we don't need to rebuild all version of JetBrains IDEs again (args to start launcher is written inside image)